web_widget_x2many_2d_matrix/README.rst

```diff
@@ -107,7 +114,14 @@ the field in the default function::
 
 Now in our wizard, we can use::
 
-<field name="task_ids" widget="x2many_2d_matrix" field_x_axis="project_id" field_y_axis="user_id" field_value="planned_hours" />
+ <field name="task_ids" widget="x2many_2d_matrix" field_x_axis="project_id" field_y_axis="user_id" field_value="planned_hours">
+     <tree>
+         <field name="task_ids"/>
```

Comment:
Are you sure that this is correct? task_ids is not a field of task_ids!

Reply:
Same error using this approach here: https://github.com/Eficent/ddmrp/pull/15 (it is v9 though)